src/Website/App_Start/OwinStartup.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using mbsoft.BrewClub.Data;
+using Microsoft.AspNet.Identity;
+using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.Owin;
+using Microsoft.Owin.Security.Cookies;
+using Owin;
+
+namespace mbsoft.BrewClub.Website
+{
+	public class OwinStartup
+	{
+		
+		public void Configuration(IAppBuilder app)
+		{
+			app.UseCookieAuthentication(new CookieAuthenticationOptions
+			{
+				AuthenticationType = DefaultAuthenticationTypes.ApplicationCookie,
+				Provider = new CookieAuthenticationProvider { OnApplyRedirect = ApplyLoginRedirect }
+			});			
+		}
+
+		private static void ApplyLoginRedirect(CookieApplyRedirectContext context)
+		{
+			var helper = new UrlHelper(HttpContext.Current.Request.RequestContext);
+			var loginUrl = helper.Action("Login", "Account");
+
+			context.Response.Redirect(loginUrl);
+		}
+	}
+}
```

Comment:
This configures the identity framework stuff.  I wouldn't mind figuring out how to use Owin properly, and allow for facebook/twitter/google logins as well, but for now, this just configures the type of authentication method, and where the user gets sent when the login fails.